### PR TITLE
[issue-51] Create blue section

### PR DESCRIPTION
**Pull request by robertnecsoiu**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--jet2holidays-portugal--aemsites.aem.live/
- After: https://issue-51--jet2holidays-portugal--aemsites.aem.page/drafts/rnecsoiu/bluesection

### CODE REVIEW

File: blocks/bluesection/bluesection.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    font-weight: 300;
+    font-size: 1rem;
+    margin: 0 0 20px;
+}
+
+.blue-section p a {
+    color: white;
+    text-decoration: underline;
+    font-style: italic;
+    font-family: 'Open Sans', sans-serif !important;
```
Comment (bstopp) [inline blocks/bluesection/bluesection.css:32]: we need to create a font-family for Open Sans, and that also includes a Fallback.

Additionally, `!important!` should not be needed in any context

**Review by bstopp — COMMENTED** [review]
This shouldn't need to be a block. It should be a section style which affects all `default content` within it's context..

**Comment by bstopp** [comment]
Converting to draft, as this isn't a block but a section styling.

File: styles/styles.css [hunk 2]
```diff
@@ -257,3 +257,34 @@ main .section.highlight {
...
   padding: 40px 0;
 }
+
+.blue-section {
+    padding: 20px;
+    background-color: #388ce1;
+    color: white;
+    text-align: center;
+    & h4 {
+        font-family: franklin-heavy-italic, sans-serif;
```
Comment (bstopp) [inline styles/styles.css:267]: This should reference a variable - not sure where `franklin-heavy-italic` is defined.

Should have a variable
```
--font-family-open-sans: 'Open Sans', 'Open Sans Fallback', Arial, sans-serif
```

Which requires a fallback definiton

File: styles/styles.css [hunk 3]
```diff
@@ -257,3 +257,34 @@ main .section.highlight {
   margin: 0;
   padding: 40px 0;
 }
+
+.blue-section {
+    padding: 20px;
+    background-color: #388ce1;
```
Comment (bstopp) [inline styles/styles.css:263]: All colors should be defined as variables for reuse.

File: styles/styles.css [hunk 4]
```diff
@@ -257,3 +257,34 @@ main .section.highlight {
...
+    & p {
+        font-family: 'Open Sans', sans-serif;
+        font-weight: 300;
+        font-size: 1rem;
+        margin: 0 0 20px;
+        & a {
+            color: white;
+            text-decoration: underline;
+            font-style: italic;
+            font-family: 'Open Sans', sans-serif !important;
```
Comment (bstopp) [inline styles/styles.css:280]: Shouldn't ever need `!important` for anything.

**Review by bstopp — CHANGES_REQUESTED** [review]

File: styles/styles.css (hunk 2 above)
Comment (robertnecsoiu) [reply styles/styles.css:270]: For this one Alex has a PR that solves fonts issues and will use the families defined in there.

**Review by bstopp — APPROVED** [review]

**Pull request merged**